pom.xml

```diff
@@ -283,6 +283,18 @@
         <version>${powermock.version}</version>
         <scope>test</scope>
       </dependency>
+
```

Comment:
Currently only servers use. But this package can be used to manually handle any special findbug cases.
Both is OK for me.